### ISSUE TITLE
Firefox Add-on page removed

### ISSUE THREAD

**Issue by tw0517tw** [refined-bitbucket/refined-bitbucket#436]
visit https://addons.mozilla.org/en-US/firefox/addon/refined-bitbucket-/ and found page 404. 
is this add-on pulled from store?

**Comment by reyronald** [comment]
Yes, it was pulled, I'll look into it this week

**Comment by ConnorSinnott** [comment]
Hey @reyronald! Any update on the Firefox plugin?

**Comment by ConnorSinnott** [comment]
Hey @reyronald is this project still active? The plugin has been absent from the Mozilla marketplace for 2 months now.

**Comment by jn-chrn** [comment]
I also would like to know, it has been 3 months now.

**Comment by ConnorSinnott** [comment]
The silence here is a bit concerning. Is the project dead? This issue is critical since it prevents Firefox usage without hacking it in through developer tools. @reyronald are you the one to speak with about setting up this plugin for Firefox, or perhaps could someone else take those reigns? Bitbucket without your extension is just... awful.

**Comment by nbrideau** [comment]
Hey, I got tired of waiting and packed 3.26.1 up for release and had Mozilla sign it. https://longreachindustries.ca/refined-bitbucket-3.26.1.xpi

**Comment by thomasmerel** [comment]
Thanks a lot @nbrideau

**Comment by nbrideau** [comment]
@thomasmerel I did it for myself and as a C dev I enjoyed nothing about the process.

**Comment by thomasmerel** [comment]
You helped at least 4 people on this issue and that, without even counting those who don't 🚀 your post 😁

**Comment by ConnorSinnott** [comment]
> @thomasmerel I did it for myself and as a C dev I enjoyed nothing about the process.

I love this post. Truly @nbrideau is the antihero we didn't deserve.

**Comment by timchaston** [comment]
@reyronald Any update on this? Do you need to bring on board more maintainers to sustain the project? I'm happy to help out if there's anything I can do (and I'm sure many other are too).

**Comment by thezanke** [comment]
Yes, I'm sure someone would take over if you no longer have to use Bitbucket and are no longer inclined to maintain this @reyronald  🙏

**Comment by RHeynsZa** [comment]
Popped @reyronald a tweet [here](https://twitter.com/RHeynsZA/status/1637862786188337162)

Lets see. Otherwise someone might as well just fork :eyes:
